src/Tools/dotnet-user-jwts/src/Commands/CreateCommand.cs

```diff
@@ -32,7 +33,7 @@ public static void Register(ProjectCommandLineApplication app)
                 );
 
             var nameOption = cmd.Option(
-                "--name",
+                "--name|-n",
```

Comment:
I think this is usually defined the other way around in the rest of the CLI, i.e.
```suggestion
                "-n|--name",
```